names.md

```diff
@@ -3,7 +3,7 @@ subl# This is a list of all the names of people working on this project
 # Format -   Full name - nickname (optional)
 Rucha Rathor
 Benjamin Chow - Ben
-Shaurav Ranjan
+Shaurav Ranjan- blue-My New Tasks
```

Comment:
You need to make your new branch based off your master branch.  You based your WT-NEW-TASK off your color change branch.